tooling/nargo_fmt/src/visitor/item.rs

```diff
@@ -146,21 +148,9 @@ impl super::FmtVisitor<'_> {
         for Item { kind, span } in module.items {
             match kind {
                 ItemKind::Function(func) => {
-                    self.format_missing_indent(span.start(), true);
-
-                    if std::mem::take(&mut self.ignore_next_node) {
-                        self.push_str(self.slice(span));
-                        self.last_position = span.end();
+                    if let ControlFlow::Break(_) = self.visit_function(span, func) {
                         continue;
```

Comment:
What does this `continue` do? It looks like after the continue this iteration is ended anyway